start

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo 'Interpretting data dictionary...'
+python bin/manage.py redcap json /Input/metadata.csv 
+echo 'generating models...'
+python bin/manage.py redcap models metadata.json -o project/models.py
+echo 'creating fixtures from data...'
+python bin/manage.py redcap fixture /Input/data.csv metadata.json project
+echo 'synchronizing database...'
+python bin/manage.py syncdb
+echo 'bootstrapping data...'
+python bin/manage.py avocado init project
+
+echo 'Loading data...'
+python /bin/manage.py loaddata fixtures.json
```

Comment:
Remove leading slash for this line and below since that will be interpreted as an absolute path.